src/services/shims.ts

```diff
@@ -293,7 +293,7 @@ namespace ts {
             const oldSnapshotShim = <ScriptSnapshotShimAdapter>oldSnapshot;
             const encoded = this.scriptSnapshotShim.getChangeRange(oldSnapshotShim.scriptSnapshotShim);
             // TODO: should this be '==='?
-            if (encoded == null) {
+            if (encoded === null) {

```

Comment:
Looking at the blame and PR where the TODO was introduced, it's clear that it should be `===`.

Link: https://github.com/Microsoft/TypeScript/pull/3529#discussion-diff-32698614R237